source/install/prod-windows-2012.rst

```diff
@@ -0,0 +1,419 @@
+
+19. When prompted, enter the password created in step 9.a.       
+
+20. At the ``mysql>`` prompt, execute the following commands to create the database and user account
+
+    .. code:: sql
+
+       CREATE DATABASE mattermost;
+       CREATE USER mmuser IDENTIFIED BY 'mmuser_password';
+       GRANT ALL ON mattermost.* TO mmuser;
+       exit
+
+21. To confirm the database and user were configured correctly
+
+    a. Connect to the MySQL server/datbase by executing ``mysql -u mmuser -p mattermost`` 
+    b. When prompted, entering ``mmuser_password``
+    c. If If successful, you will be at the ``mysql>`` prompt 
+    d. Type ``exit`` to finish
+
+Set up Mattermost Server
+------------------------
+
+1. For the purposes of this guide we will assume this server has an IP
+   address of 10.0.0.2
+
+2. `Download <https://www.mattermost.org/download/>`__ the latest Mattermost Server by opening your 
+   favorite browser and navigating to 
+   ``https://releases.mattermost.com/X.X.X/mattermost-team-X.X.X-windows-amd64.tar.gz`` 
+   where `X.X.X` is the latest Mattermost release version. For example, 3.4.0.
+
+3. Using a utility such as `7-zip <http://www.7-zip.org/>`__, extract the Mattermost Server files 
+   to ``c:\mattermost\``.
+
+4. Create the storage directory for files. We assume you will have attached a large drive for storage 
+   of images and files. For this setup we will assume the directory is located at ``c:\mattermost\data``.  
+   Your directory structure should look similar to the image below.
+
+   .. image:: ../images/windows_1_expected_directory_structure.png
+
+5. Configure Mattermost Server by editing the config.json file at ``c:\mattermost\config\config.json``.  Note that
+   you will need to open the file with Wordpad or another text-editor that supports posix-style line feeds.
+   
+   * Update database name and server in the the connection string:
+     
+     * Old: ``"DataSource": "mmuser:mostest@tcp(dockerhost:3306)/mattermost_test?charset=utf8mb4,utf8"``    
+     * New: ``"DataSource": "mmuser:mmuser_password@tcp(10.0.0.1:3306)/mattermost?charset=utf8mb4,utf8"``
+
+   .. note :: Optionally you may continue to edit configuration settings in ``config.json`` or use the 
+      System Console described in a later section to finish the configuration.
+
+6. Test the Mattermost Server
+
+   a. Open a command prompt, and execute ``cd c:\mattermost\bin`` to change your working directory
+
+   b. Execute ``platform.exe``
+   
+   c. Verify that mattermost is running and connected to the database successfully by confirmed a console 
+      log like ``Server is listening on :8065`` letting you know the service is running.
+
+      .. image:: ../images/windows_2_platform_exe_test.png
+
+   d. Stop the server for now by typing ``ctrl-c``
+
+Configure the Firewall
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. note:: This is not required if you are installing on a single server.
```

Comment:
The ".. note:" is correct for rst docs. The GitHub processor styles it differently than the Sphinx processor that's used for Mattermost docs. This is what it looks like when built using Sphinx:

![image](https://cloud.githubusercontent.com/assets/20325173/20493474/d03d57c6-afcd-11e6-9c87-5f234b647623.png)